src/components/IssueSearchAdvanced.vue

```diff
@@ -13,21 +13,56 @@ import { useSearchParams } from '@/composables/useSearchParams';
 import { flawImpacts, flawSources, flawIncidentStates, descriptionRequiredStates } from '@/types/zodFlaw';
 import { allowedEmptyFieldMapping, flawFields } from '@/constants/flawFields';
 import { affectAffectedness } from '@/types/zodAffect';
+import { useSearchStore } from '@/stores/SearchStore';
+import { useToastStore } from '@/stores/ToastStore';
 
 const props = defineProps<{
   isLoading: boolean;
 }>();
-const emit = defineEmits(['filter:save']);
+
+const { addToast } = useToastStore();
+const draftQuery = ref('');
+const draftFields = ref({});
+
+const searchStore = useSearchStore();
 const {
   facets,
+  loadAdvancedSearch,
   orderField,
   orderMode,
   query,
   removeFacet,
   submitAdvancedSearch,
   toggleSortOrder,
 } = useSearchParams();
-const { closeModal, isModalOpen, openModal } = useModal();
+
+const queryGuideModal = useModal();
+const savingSearchModal = useModal();
+
```

Comment:
Might be good to break these 2 modals into their own components since the component is getting kind of big